core/src/main/scala/filodb.core/downsample/DownsampledTimeSeriesShard.scala

```diff
@@ -74,7 +74,8 @@ class DownsampledTimeSeriesShard(rawDatasetRef: DatasetRef,
   private val partKeyIndex = new PartKeyLuceneIndex(indexDataset, schemas.part, false,
     false, shardNum, indexTtlMs,
     downsampleConfig.indexLocation.map(new java.io.File(_)),
-    if (downsampleConfig.enablePersistentIndexing) Some(new EphemeralIndexMetadataStore()) else None)
+    if (downsampleConfig.enablePersistentIndexing)
```

Comment:
Please do not remove existing code, use config to instantiate right store. We may have a different implementation of store later

Reply:
put configuration